src/pretix/control/views/vouchers.py

```diff
@@ -215,7 +215,7 @@ def get_form_kwargs(self):
     @transaction.atomic
     def form_valid(self, form):
         form.instance.event = self.request.event
-        messages.success(self.request, _('The new voucher has been created.'))
+        messages.success(self.request, _('The new voucher has been created.') + ' (%s)' % form.instance.code)
```

Comment:
nice idea! Just a quick question, I think I need to update the translation as well?

./pretix/locale/de/LC_MESSAGES/django.po
./pretix/locale/de_Informal/LC_MESSAGES/django.po
```
#: pretix/control/views/vouchers.py:218
msgid "The new voucher has been created."
msgstr "Der neue Gutschein wurde erstellt."
```